plugins/authAxios.js

```diff
@@ -2,24 +2,17 @@ import recommendation from './europeana/recommendation';
 import set from './europeana/set';
 
 export default ({ $axios, $auth, store, redirect }, inject) => {
-  const token = $auth.getToken('keycloak');
+  const token = $auth.loggedIn ? $auth.getToken('keycloak') : null;
 
   const headers = {};
   if (token) headers['Authorization'] = token;
+  else delete $axios.defaults.headers.common['Authorization'];
```

Comment:
Why is this needed?

Reply:
The token is not cleared from the browser when auth.loggedIn is false, it's still there and valid for whatever time it was set to in Keycloak (I think the value Access Token Lifespan in keycloak controls it and defaults to 5 mins) . This leads to errors, if for example you try to like an item after logout it will throw an error that the user already has a likes collection because the token is sent and it is still valid.